### PR TITLE
SEC-3.1: Add Delimiter with banner configuration

### PR DESCRIPTION
**Pull request by agvarghe-cisco**
-  Config for banner should have delimiter character "c" , adding a deviation for delimiter
    
- Adding Encoding to be used as Encoding_JSON_IETF

### CODE REVIEW

File: feature/experimental/security/aaa/kne_tests/tls_authentication_over_grpc_test/tls_authentication_over_grpc_test.go [hunk 1]
```diff
@@ -149,15 +152,14 @@ func TestAuthentication(t *testing.T) {
 				}
 			}
 			t.Log("Trying credentials with GNMI Set")
+			json_config, _ := json.Marshal(*deviations.BannerDelimiter + "message of the day" + *deviations.BannerDelimiter)
```
Comment (prinikasn) [inline feature/experimental/security/aaa/kne_tests/tls_authentication_over_grpc_test/tls_authentication_over_grpc_test.go:155]: don't use underscores in Go names; var json_config should be jsonConfig

[http://golang.org/doc/effective_go.html#mixed-caps](https://www.google.com/url?sa=D&q=http%3A%2F%2Fgolang.org%2Fdoc%2Feffective_go.html%23mixed-caps)

**Review by prinikasn — CHANGES_REQUESTED** [review]
On line 155, don't use underscores in Go names; var json_config should be jsonConfig

[http://golang.org/doc/effective_go.html#mixed-caps](https://www.google.com/url?sa=D&q=http%3A%2F%2Fgolang.org%2Fdoc%2Feffective_go.html%23mixed-caps)

File: feature/experimental/security/aaa/kne_tests/tls_authentication_over_grpc_test/tls_authentication_over_grpc_test.go (hunk 1 above)
Comment (agvarghe-cisco) [reply feature/experimental/security/aaa/kne_tests/tls_authentication_over_grpc_test/tls_authentication_over_grpc_test.go]: Changed to jsonConfig

**Review by prinikasn — COMMENTED** [review]
/gcbrun

**Review by prinikasn — APPROVED** [review]

**Pull request merged**